### PR TITLE
Adds subspec for static library workaround (2)

### PR DESCRIPTION
**Pull request by tonyxiao**
Based on the work here

https://github.com/segment-integrations/analytics-ios-integration-google-analytics/pull/13

### CODE REVIEW

**Comment by tonyxiao** [comment]
@f2prateek @chrislavender @ladanazita 
I tested with the following configurations
- Without `use_frameworks!`
  - Use default subspec
  - Use `GoogleIDFASupport` explicitly
  - Use only `Segment-GoogleAnalytics/Core`
- With `use_frameworks!`
  - Use only `Segment-GoogleAnalytics/StaticLibWorkaround`

Here's a complete example app with workaround that involves static lib and manually including Pod source files into user xcodeproj

https://github.com/segmentio/TestAppSwiftUseFrameworksStaticLib

File: Example/Podfile [hunk 1]
```diff
@@ -1,5 +1,12 @@
+## Without use_frameworks!
+
 target 'Segment-GoogleAnalytics_Example' do
+    # Use default subspec
   pod 'Segment-GoogleAnalytics', :path => '../'
+    # Use default `GoogleIDFASupport` explicitly
+#  pod 'Segment-GoogleAnalytics/GoogleIDFASupport', :path => '../'
```
Comment (f2prateek) [inline Example/Podfile]: Indentation is off here.

File: Example/Podfile [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+    # Use only `Segment-GoogleAnalytics/Core`
+#  pod 'Segment-GoogleAnalytics/Core', :path => '../'
```
Comment (f2prateek) [inline Example/Podfile]: Indentation is off here.

**Review by f2prateek — COMMENTED** [review]
Some formatting issues but looks good otherwise! 🚢

**Comment by tonyxiao** [comment]
Fixed thanks @f2prateek . Waiting for @chrislavender to confirm it's working for him and let's merge this in and close #13 after that.

File: Example/Podfile (hunk 2 above)
Comment (tonyxiao) [reply Example/Podfile]: Fixed 👍

File: Example/Podfile (hunk 1 above)
Comment (tonyxiao) [reply Example/Podfile]: Fixed 👍

**Comment by ladanazita** [comment]
Released on [1.1.5](http://cocoadocs.org/docsets/Segment-GoogleAnalytics/1.1.5/)

**Pull request merged**